nvidia/nvidia-utils/PKGBUILD

```diff
@@ -331,8 +314,9 @@ package_nvidia-open-dkms() {
   pkgdesc="NVIDIA open kernel modules - module sources"
   depends+=('dkms')
   license=('MIT AND GPL-2.0-only')
-  conflicts=('nvidia-open' 'NVIDIA-MODULE')
-  provides=('nvidia-open' 'NVIDIA-MODULE')
+  conflicts=('nvidia-open' 'NVIDIA-MODULE' 'nvidia-dkms')
+  provides=('nvidia-open' 'NVIDIA-MODULE' 'nvidia-dkms')
+  replaces=('nvidia-dkms')
```

Comment:
`conflicts` and `provides` are fine, but I'm not sure if should specify `replaces` here if we want to make sure that for Pascal/Maxwell users nvidia-580xx-dkms will replace nvidia-dkms.

Reply:
I did not think much about this yet, so this needs rework anyways

Reply:
> Don't need to explicitly conflict and provide `nvidia-dkms` because that provides `NVIDIA-MODULE`. Are we going to have the 580 drivers in repo? If yes then we should first migrate closed users to that package. Regardless of that, IMO we should be rid of the `-open` suffix, since theres only one variant now.

So, the nvidia-closed module is still provided by NVIDIA but the testing is bare minimum. This driver could be only used to disable GSP Firmware.